docs/Users_Guide/overview.rst

```diff
@@ -572,7 +572,7 @@ METplus Components Python Requirements
       (scikit-learn)
       <../generated/model_applications/marine_and_cryosphere/GridStat_fcstRTOFS_obsSMAP_climWOA_sss.html>`_
   * - scipy
```

Comment:
This is listed as 1.8.1 in calcpy and plotpy nco_requirements.txt and 1.8.1 in plotpy requirements, but 1.8.0 in calcpy requirements.txt.  Should we update calcpy requirements